include/task.h

```diff
@@ -358,8 +358,8 @@ typedef enum
  * task. h
  * <pre>
 * TaskHandle_t xTaskCreateStatic( TaskFunction_t pxTaskCode,
- *                               const char *pcName,
- *                               uint32_t ulStackDepth,
+ *                               const char * const pcName,
+ *                               const configSTACK_DEPTH_TYPE ulStackDepth,
```

Comment:
This function always used uint32_t so this change is not consistent with configSTACK_DEPTH_TYPE defaulting to uin16_t, as per my previous comment.